webpack/stories/index.js

```diff
@@ -1,104 +1,9 @@
-import React from 'react';
+import { configure } from '@storybook/react';
 
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
+const req = require.context('../assets/javascripts/react_app', true, /.stories.js$/);
 
-require('../assets/javascripts/bundle');
-require('../../app/assets/javascripts/application');
-require('../../app/assets/stylesheets/base.scss');
```

Comment:
I'm not sure if these three lines are necessary. I don't see any changes when they're removed.

Reply:
the idea behind it was to reuse the current foreman asset pipline(rails) js and css. AFAIR they are used in a few places. (this would allow a difference between the stories and the app)